src/main/java/org/folio/edge/oaipmh/MainVerticle.java

```diff
@@ -22,6 +24,8 @@ public Router defineRoutes() {
     if (reqTimeoutMs == Constants.DEFAULT_REQUEST_TIMEOUT_MS) {
       reqTimeoutMs = 7200000;
     }
+    log.debug("reqTimeoutMs: {}", reqTimeoutMs);
+    log.debug("okapiURL: {}", okapiURL);
```

Comment:
We don't need to log okapi url. It is configurable parameter and application will fail if it will be incorrect.